ethereum/PFP.sol

```diff
@@ -64,19 +64,27 @@ library MerkleProof {
 contract PFP is ERC721A, Ownable, VRFConsumerBase {
     using Strings for uint256;
     
-    uint256 public MAX_TOKENS = 20000;
-    string public baseExtension = ".json";
-    string private _baseURIextended;
+    uint256 public constant MAX_TOKENS = 20000;
+    string public constant BASE_EXTENSION = ".json";
+    string public _baseURIextended;
     address public _passAddress;
     bool public revealed = false;
     string public notRevealedUri = "";
     uint256 public randStartPos;
-    bytes32 public vrfKeyHash;
+    bytes32 public immutable vrfKeyHash;
     bytes32 public request_id;
-    IEPSDelegationRegister public EPS;
+    IEPSDelegationRegister public immutable EPS;
     bytes32 public snapshotMerkleRoot;
-    mapping (address => uint256) private _claimedAmount;
     
+    mapping (address => uint256) public _claimedAmount;
```

Comment:
I don't see any reason for this to be private, as it might be useful to review this for people that aren't sure how many they have left to mint?